cli/src/cmd/build.rs

```diff
@@ -88,20 +88,10 @@ pub fn collect_crate_metadata() -> Result<CrateMetadata> {
     })
 }
 
-/// Invokes `cargo build` in the current directory.
-///
-/// Currently it assumes that user wants to use `+nightly`.
-fn build_cargo_project() -> Result<()> {
-    // We also assume that the user uses +nightly.
+fn exec_cargo(command: &str, args: &[&'static str]) -> Result<()> {
     let output = Command::new("cargo")
-        .args(&[
-            "+nightly",
```

Comment:
I'm guessing that removing +nightly was intentional.
Should we now use `cargo +nightly contract build --abi` as the correct build command?

Reply:
Actually that wasn't intentional, I've added it back. Not sure whether running `cargo +nightly contract build --abi` would mean the invoked `cargo` commands also receive the flag.

The handling of this is improved in https://github.com/paritytech/ink/pull/197, which I hope will be merged first.

Reply:
If you run `cargo +nightly contract build --abi` it compiles correctly. I don't use nightly as default and it only builds if I use the `+nightly` flag. The final question will be if we hide it as part of the build command or it should be handled explicitly?